modules/ROOT/pages/bundling-plugins.adoc

```diff
@@ -7,16 +7,20 @@
 include::partial$module-loading/bundling-ref-example.adoc[]
 :!editorcomponent:
 
+== Example 1
+
 The following table shows examples of the syntax used to bundle the following plugin.
 
 Required files for an example plugin:
 
-....
+.when using `content.css`
+[source, js]
+----
 ./plugins/example/content.css
```

Comment:
We wouldn't be able to include the content.css with a require that is what the bundle feature is supposed to fix. Not sure how we would want to describe Example 1 really with 6.8 Example 2 is the way to bundle.

I wonder if we are doing this overly complicated maybe we should just include a code sample for pre 6.8 and one for post 6.8 of imports the pre not including the css files and the post including the new css files that the plugins have.

I would imagine that a developer would just copy that and remove any plugins that they don't currently own since those would error out if they don't have them.

Reply:
@spocke as discussed, we have removed the original example 1, as we want to move to the new `bundling process` for users. 

Can you please:
- [ ] verify and update any errors for the now, `new` example before we `merge` this. @spocke